pom.xml

```diff
@@ -211,7 +211,7 @@ under the License.
       <dependency>
         <groupId>jakarta.platform</groupId>
         <artifactId>jakarta.jakartaee-bom</artifactId>
-        <version>9.1.0</version>
+        <version>10.0.0</version>
```

Comment:
The latest appears to be 11.0.0. I don't know the differences between the two, though. I imagine Jetty is tied to a specific version, but I also wonder if jetty-servlet would have used 11, and the reason you went with 10 was because you're using jetty-ee10-servlet instead?